### ISSUE TITLE
推荐一本关于 Bayesian Probability 的书 open source interactive book

### ISSUE THREAD

**Issue by lxm1117** [neuronstar/general-discussions#2]
https://github.com/lxm1117/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers。 这本即有一些模型的内容也侧重相关的编程。而且有一些ipython notebook 的例子。 书目如下。
Prologue: Why we do it.

Chapter 1: Introduction to Bayesian Methods Introduction to the philosophy and practice of Bayesian methods and answering the question, "What is probabilistic programming?"

Chapter 2: A little more on PyMC We explore modeling Bayesian problems using Python's PyMC library through examples. How do we create Bayesian models?

Chapter 3: Opening the Black Box of MCMC We discuss how MCMC, Markov Chain Monte Carlo, operates and diagnostic tools.

Chapter 4: The Greatest Theorem Never Told We explore an incredibly useful, and dangerous, theorem: The Law of Large Numbers.

Chapter 5: Would you rather lose an arm or a leg? The introduction of loss functions and their (awesome) use in Bayesian methods.

Chapter 6: Getting our prior-ities straight Probably the most important chapter. We examine our prior choices and draw on expert opinions craft priors.

Chapter X1: Bayesian methods in Machine Learning and Model Validation We explore how to resolve the overfitting problem plus popular ML methods.

Chapter X2: More PyMC Hackery We explore the gritty details of PyMC.
